packages/flet/lib/src/controls/navigation_rail.dart

```diff
@@ -154,6 +154,9 @@ class _NavigationRailControlState extends State<NavigationRailControl>
                             parentAdaptive: widget.parentAdaptive)
                         : Text(label));
               }).toList());
+
+          return constrainedControl(
```

Comment:
why not `baseControl` here?

Reply:
I pushed an update rn.
Python end uses constrained control.

Reply:
OK, right. Any particular reason you moved `constrainedControl()` inside LayoutBuilder?
NavigationRail is basically unchanged.

Reply:
Because we were using `constrainedControl` twice (for `LayoutBuilder` and `Rail`) - unnecessary.

Reply:
Sorry if I'm missing something, but there is only one `constrainedControl()` call right now in https://github.com/flet-dev/flet/blob/main/packages/flet/lib/src/controls/navigation_rail.dart#L161

Reply:
Yes, I removed the second call because I found it unnecessary.
what's your point actually?

Reply:
This is `main` right now:

In `rtl` branch you first added a second `constrainedControl()` and then removed, but in different place. As a result `constrainedControl()` in this PR has been moved inside `LayoutBuilder`.

<img width="634" alt="image" src="https://github.com/user-attachments/assets/34a33cf0-2247-44da-a693-f0c0fbd37721">

Reply:
Ah, I get your point.
Should we revert this?

Reply:
I would, can't wrap my head now to understand if it changes the control behaviour :)